draft-ietf-quic-transport.md

```diff
@@ -860,39 +860,93 @@ different IP or port at either endpoint, due to NAT rebinding or mobility, as
 described in {{migration}}.  Finally a connection may be terminated by either
 endpoint, as described in {{termination}}.
 
+
+## Matching Packets to Connections
+
+Incoming packets are classified on receipt.  Packets can either be associated
+with an existing connection, be discarded, or - for servers - potentially create
+a new connection.
+
+A packet that can be identified as belonging to an existing connection is
+handled according to the current state of that connection.  Packets are
+associated with established connections using connection ID if it is present;
+this might include connection IDs that were advertised using NEW_CONNECTION_ID
+({{frame-new-connection-id}}).  Packets without connection IDs and long-form
+packets for connections that have incomplete cryptographic handshakes are
+associated using the tuple of source and destination IP addresses and ports.
+
+A packet that uses the short header form could be associated with an existing
+connection with an incomplete cryptographic handshake.  Such a packet could be a
+valid packet that has been reordered with respect to the long-form packets that
+will complete the cryptographic handshake.  These packets SHOULD be buffered in
+anticipation of the handshake completing.
+
+Buffering ensures that data is not lost, which improves performance; conversely,
+discarding these packets could create false loss signals for the congestion
+controllers.  However, limiting the number and size of buffered packets might be
+needed to prevent exposure to denial of service.
+
+For clients, all packets that cannot be associated with a connection MUST be
+discarded.  Discarded packets MAY be logged for diagnostic or security purposes.
+
+For servers, packets that aren't associated with a connection potentially create
+a new connection.  However, only packets that use the long packet header and
+packets that meet the minimum size defined for the protocol version can be
+initial packets.  A server MUST discard packets that use the short header form,
+or packets that are smaller than the smallest minimum size for any version that
+the server supports.
+
+This version of QUIC defines a minimum size for initial packets of 1200 octets.
+Versions of QUIC that define smaller minimum initial packet sizes need to be
+aware that initial packets will be discarded by servers that only support
+versions with larger minimums.  Clients that support multiple QUIC versions can
+avoid this problem by ensuring that they increase the size of their initial
+packets to the largest minimum size across all of the QUIC versions they
+support.
```

Comment:
You need to provide the appropriate guidance for servers of accepting the minimum size, regardless of version.

Suppose we have versions 1, 2, 3 with minimum sizes 1200, 1201 and 1202 respectively. A client which supports (1, 2) sends a packet of size 1201. A server which supports (2, 3) and doesn't recognize 1 needs to accept [1, 1201] and send VN even though [3, 1201] would not be acceptable